test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/AcceptHeaderParserTest.cs

```diff
@@ -49,6 +51,60 @@ public void ParseAcceptHeader_ParsesSimpleHeaderWithMultipleValues()
         }
 
         [Fact]
+        public void ParseAcceptHeader_ParsesSimpleHeaderWithMultipleValues_InvalidFormat()
+        {
+            // Arrange
+            var header = "application/json, application/xml,;q=0.8";
+            var expectedMediaTypes = new List<MediaTypeSegmentWithQuality> {
+                new MediaTypeSegmentWithQuality(new StringSegment("application/json"),1.0),
+                new MediaTypeSegmentWithQuality(new StringSegment("application/xml"),1.0),
+            };
+
+            // Act
+            var mediaTypes = AcceptHeaderParser.ParseAcceptHeader(new List<string> { header });
+
+            // Assert
+            Assert.Equal(expectedMediaTypes, mediaTypes);
+        }
+
+        public static TheoryData<string[], string[]> ParseAcceptHeaderWithInvalidMediaTypesData =>
+            new TheoryData<string[], string[]>
+            {
+                { new [] { ";q=0.9" }, new string[] { } },
+                { new [] { "/" }, new string[] { } },
+                { new [] { "*/" }, new string[] { } },
+                { new [] { "/*" }, new string[] { } },
+                { new [] { "/;q=0.9" }, new string[] { } },
+                { new [] { "*/;q=0.9" }, new string[] { } },
+                { new [] { "/*;q=0.9" }, new string[] { } },
+                { new [] { "/;q=0.9,text/html" }, new string[] { "text/html" } },
+                { new [] { "*/;q=0.9,text/html" }, new string[] { "text/html" } },
+                { new [] { "/*;q=0.9,text/html" }, new string[] { "text/html" } },
+                { new [] { "img/png,/;q=0.9,text/html" }, new string[] { "img/png", "text/html" } },
+                { new [] { "img/png,*/;q=0.9,text/html" }, new string[] { "img/png", "text/html" } },
+                { new [] { "img/png,/*;q=0.9,text/html" }, new string[] { "img/png", "text/html" } },
+                { new [] { "img/png, /;q=0.9" }, new string[] { "img/png", } },
+                { new [] { "img/png, */;q=0.9" }, new string[] { "img/png", } },
+                { new [] { "img/png, /*;q=0.9" }, new string[] { "img/png", } },
```

Comment:
Can you add a scenario where a mediaa type has a valid quality factory mixed along with one that's invalid?